app/templates/releases/canary.hbs

```diff
@@ -19,3 +19,21 @@ npm install --save-dev https://s3.amazonaws.com/builds.emberjs.com{{model.canary
   project=model.emberData
   channel="canary"}}
 </div>
+
+<h2>What is canary?</h2>
+
+<p>
+  "Canary" is whatever code is on the main branch of an Ember project.
+  It is unreleased, and is not recommended for production or new learners.
+  Often, features in canary are a work-in-progress.
+  They may change substantially or be removed before being released,
+  and in rare cases, they may never be released.
```

Comment:
"they" in "they may never be released" is a bit ambiguous.

Reply:
I expanded the sentence removing "They" to be more explicit